<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortrayalParametersDlg</class>
 <widget class="QDialog" name="PortrayalParametersDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portrayal parameters</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>729</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="shallowContourLabel">
            <property name="text">
             <string>Shallow contour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="safetyContourLabel">
            <property name="text">
             <string>Safety contour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="deepContourLabel">
            <property name="text">
             <string>Deep contour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="safetyDepthLabel">
            <property name="text">
             <string>Safety depth</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="shallowContour"/>
          </item>
          <item>
           <widget class="QLineEdit" name="safetyContour"/>
          </item>
          <item>
           <widget class="QLineEdit" name="deepContour"/>
          </item>
          <item>
           <widget class="QLineEdit" name="safetyDepth"/>
          </item>
          <item>
           <spacer name="verticalSpacer2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutCheckBox">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="colorMode">
          <property name="text">
           <string>Colour mode - 4 colour (rather than 2 colour)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolizedBoundaries">
          <property name="text">
           <string>Symbolized boundaries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="paperChartSymbols">
          <property name="text">
           <string>Paper chart symbols</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="realSectorLegLength">
          <property name="text">
           <string>Show real sector leg length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="shallowPattern">
          <property name="text">
           <string>Shallow pattern</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="dangersInShallowWaters">
          <property name="text">
           <string>Show dangers in shallow waters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="usePeriodicAttributes">
          <property name="text">
           <string>Use periodic attributes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="useSCAMINSCAMAX">
          <property name="text">
           <string>Use SCAMIN/SCAMAX attributes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="antialiasing">
          <property name="text">
           <string>Antialiasing</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>235</red>
            <green>235</green>
            <blue>235</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>235</red>
            <green>235</green>
            <blue>235</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>64</red>
            <green>64</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>My viewing groups</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="restoreCurrent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Restore current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setMyViewingGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="myViewingGroups">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>230</red>
               <green>230</green>
               <blue>230</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveMyViewingGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteMyViewingGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>235</red>
            <green>235</green>
            <blue>235</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>235</red>
            <green>235</green>
            <blue>235</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>230</red>
            <green>230</green>
            <blue>230</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="title">
       <string>Viewing groups for</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="HighlightedText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="HighlightedText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>230</red>
               <green>230</green>
               <blue>230</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="HighlightedText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonApply</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setMyViewingGroup</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>OnSetViewingGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveMyViewingGroup</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>OnSaveViewingGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restoreCurrent</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>OnRestoreViewingGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteMyViewingGroup</sender>
   <signal>released()</signal>
   <receiver>PortrayalParametersDlg</receiver>
   <slot>OnDeleteViewingGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
  <slot>OnApply()</slot>
  <slot>OnSetViewingGroup()</slot>
  <slot>OnSaveViewingGroup()</slot>
  <slot>OnRestoreViewingGroup()</slot>
  <slot>OnDeleteViewingGroup()</slot>
 </slots>
</ui>
